plugins/alignments/src/PileupRenderer/configSchema.ts

```diff
@@ -55,6 +55,11 @@ export default ConfigurationSchema(
       description: 'remove spacing between features',
       defaultValue: false,
     },
+    largeInsertionIndicatorScale: {
+      type: 'number',
+      description: 'scale at which to draw the large insertion indicators',
```

Comment:
maybe add `(units: bases / pixel)` or similar to clarify what the number is

Reply:
:+1: